Parser/listnode.c

```diff
@@ -15,19 +16,21 @@ PyNode_ListTree(node *n)
     listnode(stdout, n);
 }
 
-static int level, atbol;
-
 static void
 listnode(FILE *fp, node *n)
 {
-    level = 0;
-    atbol = 1;
+    PyInterpreterState *interp = _PyInterpreterState_GET_UNSAFE();
+
+    interp->parser_data.listnode_data.level = 0;
```

Comment:
I was going to point to the advantage of having stand-alone structs for places like this (so there isn't so much going on with each line).  However, I actually kind of like how the line captures the indirection clearly. :)